res/app/device-list/details/device-list-details-directive.js

```diff
@@ -277,6 +279,20 @@ module.exports = function DeviceListDetailsDirective(
         }
       })()
 
+      // check if new column needs a new scope or not
+      // and build accordingly
+      function buildColumn(col, device) {
```

Comment:
yeah, I will try to figure out a new way in which I don't need to create new scope or create only when required.

Never thought that fake devices can be used for performance testing. Thanks for the advice.